src/ComponentPage.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+
+import './ComponentPage.scss'
+
+class ComponentPage extends Component {
+  renderSidebarLink(link, index) {
+    return (
+      <a
+        href={link.url}
+        key={index}
+        role='listitem'     
+        className={`mdc-list-item ${link.className} ${link.active ? 'sidebar-active' : ''}`}>
+          {link.content}
+       </a>
+    );
+  }
+
+  renderSidebar(activeLink) {
+    const links = [{
+      content: 'Buttons',
+      url: '/buttons.html',
+      className: 'sidebar-buttons',
+      active: activeLink === 'Buttons',
+    }, {
+      content: 'Cards',
+      url: '/card.html',
+      className: 'sidebar-cards',
+      active: activeLink === 'Cards',
+    }];
+
+    return(
+      <section className='sidebar mdc-layout-grid__cell mdc-layout-grid__cell--span-2'>
+        {links.map(this.renderSidebarLink)}
+      </section>
+    );
+  }
+
+  renderResource(title, imageSource, url) {
+    return (
+      <a href={url} role='listitem' className='mdc-list-item' target='_blank'>
+        <span className='mdc-list-item__graphic'><img src={imageSource} className='resources-icon'/></span>
```

Comment:
There is a compilation warning due to this img missing an alt tag. We should fix that, maybe by just feeding the title to the alt text as well, or "<title> icon"